src/components/fields/ObjectField.js

```diff
@@ -117,11 +117,13 @@ class ObjectField extends Component {
         {title ? <TitleField
                    id={`${idSchema.$id}__title`}
                    title={title}
-                   required={required} /> : null}
+                   required={required}
+                   formContext={formContext}/> : null}
         {schema.description ?
           <DescriptionField
             id={`${idSchema.$id}__description`}
             description={schema.description}
+            formContext={formContext}
           /> : null}
```

Comment:
We should try to stay consistent, it seems everywhere else we append the self-closing tag bit on the same line.